src/views/Unfollow/index.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory, useLocation } from 'react-router-dom';
+import { useMutation, useQuery } from '@apollo/client';
+
+import Alert from 'components/shared/Alert';
+import useMessage from 'hooks/useMessage';
+import GetBasics from 'queries/Basics/GetBasics';
+import {
+  GetBasics as GetBasicsType,
+  GetBasicsVariables
+} from 'queries/Basics/types/GetBasics';
+import DeletePlanFavorite from 'queries/Favorite/DeletePlanFavorite';
+import { DeletePlanFavoriteVariables } from 'queries/Favorite/types/DeletePlanFavorite';
+
+type UnfollowProps = {
+  children: React.ReactNode;
+};
+
+const UnfollowWrapper = ({ children }: UnfollowProps) => {
+  const history = useHistory();
+  const { t } = useTranslation('modelPlan');
+  const { pathname, search } = useLocation();
+  const { showMessageOnNextPage } = useMessage();
```

Comment:
Gary, I'm not sure having a wrapper for a single route is the right method.  As is, this wrapper computes on every route change, as will changes to all the `useEffect` dependencies.  We only need to compute this logic if the router hits `/unfollow`.  This logic should be wrapped in a component that is only trigger within the `BrowserRouter`.  Wrappers like this are only necessary when you need to check states on every route in the app, like NDA and Subscriptions.  I think `/unfollow' should have its own designated route in our `App` component and only be called if the route is hit